packages/vip-go/services/logService.js

```diff
@@ -23,7 +22,6 @@ const getService = (namespace) => {
   let service = defaultService;
   let sentryConfig = {};
   let Sentry;
```

Comment:
I'd maybe suggest still putting this outside the function body, just in case.

Reply:
Oh right.